novawallet/Common/Storage/WalletsUpdateMediator.swift

```diff
@@ -135,7 +135,7 @@ final class WalletUpdateMediator {
             // we want to change selected wallet if current one is removed or revoked as proxied
 
             let newUpdatedWallets = changes.newOrUpdatedItems.map { wallet in
-                if wallet.isSelected, wallet.info.proxy()?.status == .revoked {
+                if wallet.isSelected, wallet.info.proxy?.status == .revoked {
```

Comment:
Just noticed that we didn't move `WalletsUpdateMediator` to delegation notion yet but it should be done in terms of delegated account detection logic